pkg/agent/openflow/client.go

```diff
@@ -326,6 +328,11 @@ type Client interface {
 	// installed by InstallMulticastGroup.
 	UninstallMulticastGroup(groupID binding.GroupIDType) error
 
+	// InstallMulticastForwardInboundGroup installs group with two buckets.
+	// One Bucket sets the traffic to multicast OVS pipeline.
+	// Another bucket outputs the traffic to the host interface.
```

Comment:
```suggestion
	// The other outputs the traffic to the host interface.
```

Reply:
Add "This group only works on the packet that enters OVS from uplink" ?

Reply:
updated